extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -578,15 +609,16 @@ export class SafeframeHostApi {
    * @param {number} height
    * @param {number} width
    * @param {string} messageType
-   * @param {boolean=} optIsCollapse
+   * @param {boolean=} optIsReducing True for collapse & shrink requests.
    */
-  resizeAmpAdAndSafeframe(height, width, messageType, optIsCollapse) {
+  resizeAmpAdAndSafeframe(height, width, messageType, optIsReducing) {
+    const isCollapsed = messageType == SERVICE.COLLAPSE_RESPONSE;
```

Comment:
Do we need optIsReducing at all?  Can't we always just look at the messageType?

Reply:
done